### ISSUE TITLE
User list not displaying correctly

### ISSUE THREAD

**Issue by Michael-073** [PiranhaCMS/Piranha#467]
Hello,
I have a rather large amount of users on my website, but when i try to search for one, it does not appear in the list. I know they exist because if I can get the entire list to load by going to the group tab they are assigned to, I can see the table row.
I did some digging and debugging and i think I may have found the culprit. On line 427 of the jquery.manager.js file
`for (var i = 0, il = is.length; i < il && i < self.maxVisibleItemsCount; i++) {`
The last part of the conditional doesn't look right to me. `i < self.maxVisibleItemsCount`
From what I could gather from the code, this would stop searching after it reaches the max visible items number. Even if there were more items in the list that were found they wouldn't be displayed.
I realize this may not be your code, but it is rather old, and I doubt the original file is still supported by the main author.

If I wanted to change the file myself, I would need to modify the file from the Piranha source and then recompile it correct?

**Comment by tidyui** [comment]
Yes there is a limitation in list.js which is used for sorting, but I added a parameter to resolve it for the page list:

https://github.com/PiranhaCMS/Piranha/blob/master/Core/Piranha/Areas/Manager/Views/Page/Index.cshtml#L115

The easy solution in your case would be to copy the User list view from here:

https://github.com/PiranhaCMS/Piranha/blob/master/Core/Piranha/Areas/Manager/Views/Settings/User.cshtml

and put it locally in your project in the exact same location, this will override the view embedded in the dll.  Then you can set the maxVisibleItemsCount in your local view.

Regards

**Comment by Michael-073** [comment]
Thank you for the help. Although I don't think that really solves the issue, since it will still not show anything above the set amount, and increasing the number even higher can cause memory issues and slow down to occur on the page.
I decided to just copy the jquery.manger.js file into my directory and changed the reference from the embedded file to the one I copied. Then just added a variable in the method to act as the count of each found item. Then switch `i < self.maxVisibleItemsCount` to use my variable in place of i
